3p/ampcontext.js

```diff
@@ -41,30 +41,20 @@ export class AmpContext extends IframeMessagingClient {
   constructor(win) {
     super(win);
     this.setupMetadata_();
+    this.getHostWindow();
```

Comment:
yes. right now, it is only called from the event listener in the parent class, so it will only fire getHostWindow the first time we get a message back. we need the hostWindow to send a message out, however, which is before we ever get a message back.